modules/ossm-control-plane-verify.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-control-plane-verify_{context}"]
+
+= Verifying the control plane installation
+
+Verify that the control plane installation completed successfully.
+
+.Procedure
+
+[NOTE]
+====
+The name of the resource is `istio-installation`.
+====
+
+. Run this command to determine if the Operator finished deploying the control plane:
++
+----
+$ oc get servicemeshcontrolplane/basic-install -n istio-system --template='{{range .status.conditions}}{{printf "%s=%s, reason=%s, message=%s\n\n" .type .status .reason .message}}{{end}}'
+----
++
+When the control plane installation is finished, the output is similar to the following:
++
+----
+Installed=True, reason=InstallSuccessful, message=%!s(<nil>)
```

Comment:
Output should look like:

Installed=True, reason=InstallSuccessful, message=%!s(<nil>)

Reconciled=True, reason=InstallSuccessful, message=%!s(<nil>)

Reply:
Fixed.